src/test/java/fashionable/simba/yanawaserver/rating/domain/RatingTest.java

```diff
@@ -0,0 +1,49 @@
+package fashionable.simba.yanawaserver.rating.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class RatingTest {
+
+    Rating rating;
+
+    @BeforeEach
+    void setUp() {
+        rating = new Rating(1L, 1L, 1L, new RatingScore(BigDecimal.valueOf(3.0)), MannerTemperatureType.EXCELLENT, 1L, "후기");
+    }
+
+    @Test
+    @DisplayName("평가를 생성한다.")
+    void ratingTest() {
+        org.junit.jupiter.api.Assertions.assertAll(
```

Comment:
static import를 추가하는게 좋아보여요 ☺️

```suggestion
        assertAll(
```